c_glib/plasma-glib/client.cpp

```diff
@@ -431,6 +431,24 @@ gplasma_client_refer_object(GPlasmaClient *client,
   }
 }
 
+/**
+ * gplasma_client_disconnect:
+ * @client: A #GPlasmaClient.
+ * @error: (nullable): Return location for a #GError or %NULL.
+ *
+ * Returns: %TRUE on success, %FALSE if there was an error.
+ *
+ * Since: 0.12.0
+ */
+gboolean
+gplasma_client_disconnect(GPlasmaClient *client,
+                          GError **error)
+{
+  auto plasma_client = gplasma_client_get_raw(client);
+  auto status = plasma_client->Disconnect();
```

Comment:
We need to keep "disconnected" information.
If users don't call `gplasma_client_disconnect()` explicitly, we call `Disconnect()` on `fainalize()`.

```cpp
if (garrow_error_check(...)) {
  priv->disconnected = true;
  return TRUE;
} else {
  return FALSE;
}
```

Reply:
I’ll keep "disconnected” information. If `priv->disconnected` is true, I will not call `Disconnected` on finalize().